### ISSUE TITLE
Sigmoid applied twice when calculating cost?

### ISSUE THREAD

**Issue by balvisio** [cansyl/DEEPred#5]
Hi all,

I was looking at `4_layer_train.py` and I noticed that the cost function uses the TF function: `tf.nn.sigmoid_cross_entropy_with_logits`

However, the `logits` argument is `A3` which is the output of the sigmoid activation (`A3 = tf.sigmoid(Z3)`) 

IIUC, the point of the `sigmoid_cross_entropy_with_logits` functions is that it has a sigmoid so the model shouldn't have any activation function on the last layer. Thus, I think it should be `logits=Z3`.

Let me know if I am missing something.

**Comment by ahmetrifaioglu** [comment]
Hi @balvisio ,

Thanks for pointing out this issue. Yes, you are right. The second sigmoid is unnecessary, as you pointed out. I have made the necessary changes in the files. 

I would like to point out that the current implementation is not supported anymore. We are now creating a new version of DEEPred.  Hopefully, it will be out this summer.

**Comment by balvisio** [comment]
Great. Thanks for the clarification.
